package.json

```diff
@@ -13,10 +13,8 @@
       "text"
     ],
     "exclude": [
-      "**/**/**/swaggerDefinition.js",
-      "**/**/**/index.js",
-      "**/**/**/sendResult.js",
-      "**/**/**/validator.js"
+      "**/**/utils/*.js",
+      "**/**/**/index.js"
```

Comment:
you don't need this because `at line 17` all `index.js` files were already added